exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServer.java

```diff
@@ -52,7 +57,10 @@ public final class PrometheusHttpServer implements Closeable, MetricReader {
 
   private static final DaemonThreadFactory THREAD_FACTORY =
       new DaemonThreadFactory("prometheus-http");
+  private static final Logger LOGGER = Logger.getLogger(PrometheusHttpServer.class.getName());
 
+  private final Set<String> allConflictHeaderNames =
+      Collections.newSetFromMap(new ConcurrentHashMap<>());
```

Comment:
interesting. Is this the best way to create a concurrent hash set, which doesn't actually exist?